cardano.json

```diff
@@ -1881,17 +1881,16 @@
 
   , "Utxo":
     { "title": "Utxo"
-    , "type": "array"
-    , "additionalItems": false
-    , "items":
-      { "type": "array"
-      , "additionalItems": false
-      , "minItems": 2
-      , "maxItems": 2
-      , "items":
-        [ { "$ref": "cardano.json#/definitions/TransactionOutputReference" }
-        , { "$ref": "cardano.json#/definitions/TransactionOutput" }
-        ]
+    , "type": "object"
+    , "propertyNames":
+      { "pattern": "^[0-9a-f]{64}#[0-9]+$" }
```

Comment:
This kind of key encoding multiple information at once has shown to be quite difficult to parse for clients down the line so that is something I've been trying to avoid in the schema.

Reply:
Also, if changed here, it should be changed consistently across all occurrences of  `TransactionOutputReference` which then becomes a little awkward.